recipe/meta.yaml

```diff
@@ -31,8 +36,13 @@ source:
     folder: plugins/access-control
 
 build:
-  number: 0
+  number: 1
   skip: True   # [unix]
+  string: h{{ PKG_HASH }}_log_{{ PKG_BUILDNUM }}  # [nsis_variant == 'log_enabled']
+  track_features:
+    - nsis_log_enabled  # [nsis_variant == 'log_enabled']
+
+requirements:
 
```

Comment:
Do we need an empty **build** section too?

Reply:
I think you're right, if I 'm reading our standards correctly. It should at least have something.